client/app/abci.go

```diff
@@ -27,7 +27,7 @@ func newABCIWrapper(
 	app abci.Application,
 	finalizeCallback postFinalizeCallback,
 	multiStoreProvider multiStoreProvider,
-) *abciWrapper {
+) abci.Application {
```

Comment:
nit: maybe not using the interface for constructor

Reply:
Good point. Changed in [004b83d](https://github.com/piplabs/story/pull/482/commits/004b83dbbff5cb3df9b25104f79af6f1a36559ae).